javascript/async-apis/APIs.md

```diff
@@ -203,6 +203,14 @@ If all goes well, you should see a new image on the page every time you refresh!
 
 If you've gotten lost along the way, check out [this jsbin project](http://jsbin.com/canofar/edit?html,output). Besides the glorious styling, this is what your version should look like.
 
+### APIs: Best Practices
+
+For now, we will not be using any APIs that are vulnerable to attacks like key theft. If you are using APIs from sources outside of our recommended curriculum, you should be aware of the potential security risks of publishing your API keys on github.
+
+1. Learn about why you should not expose your API keys [here](https://www.programmableweb.com/news/why-exposed-api-keys-and-sensitive-data-are-growing-cause-concern/analysis/2015/01/05). The article references [My $2375 Amazon EC2 Mistake](https://web.archive.org/web/20150102022540/http://www.devfactor.net/2014/12/30/2375-amazon-mistake/) written by Andrew Hoffman after he had accidentally pushed code to GitHub that included his AWS API keys.
+2. Read about why and how to secure your API keys [here](https://nordicapis.com/hiding-and-securing-your-api-keys/). This is just one method that details how to secure your key with `dotenv`.
```

Comment:
This doesn't apply to frontend code. If you hit the API directly from the frontend, the key has to be there, dotenv or not. 

This could be misleading to students. 

We should talk about the *only* way to secure a key is to keep it on the backend in a file that is not committed to git. Which we will get to later in the course.

Reply:
That makes sense. I think we can remove #2. Let me know if you have any suggestions for the rest of this. Thanks!